tests/queries/0_stateless/00754_distributed_optimize_skip_select_on_unused_shards.reference

```diff
@@ -1,16 +1,16 @@
 OK
 OK
 1
-FAIL
+OK
```

Comment:
Arguably looks weird. It corrects the garbage changes done to this and the corresponding "_with_prewhere.reference" file in #11923. This other PR made five test statements produce "Positional arguments are not supported" exceptions. But not the test statements were fixed, the expected results were adjusted... This meant that the test didn't test what it was supposed to test for three years. With my PR, clickhouse-client now behaves correctly, and I am reverting the expected results back to their actually expected values.